README.md

```diff
@@ -2,6 +2,32 @@
 
 Ever wanted to generate new Dutch slang? Look no further! This application makes use of Recurrent Neural Networks trained on a comprehensive database of Dutch slang to generate new slang. Some words may immediately bring up a visceral feeling of what they mean, and this application allows you to provide the community with your interpretation of what the newly created slang means.
 
+## Examples
+We have several pre-trained models available to generate novel words from multiple classes.
+
+### Straattaal
+- rampa
+- joeko
+- opjo
+- oggor
+- zittie
+
+### Plaatsnamen
+- Boschem
+- De Heel
```

Comment:
This one is just so creative lol. I for one would definitely believe that this was a town in Limburg or het Rivierengebied